tests/Fixtures/Integration/set/@PER-CS2.0.test

```diff
@@ -2,3 +2,5 @@
 Integration of @PER-CS2.0.
 --RULESET--
 {"@PER-CS2.0": true}
+--REQUIREMENTS--
+{"php": 80000}
```

Comment:
nope.
if you need to test specific part of ruleset that requires specific engine, test that part of ruleset under specific engine. 

now, you said that the ruleset is not tested <8 at all.

eg ` @PSR12_php71.test` test, as example

Reply:
I'm not sure I understand. There also is a ruleset test for PHP 7.4 specifically.

Reply:
we shall test as much as possible on lowest PHP version, and on high PHP version check only the "new part".

instead, the PR test as much as possible on "high PHP", and on low PHP the "old part".

Reply:
with new commits, I see it solved 👍🏻